src/lib.rs

```diff
@@ -192,8 +198,10 @@ impl Network {
         self.range
     }
 
-    pub fn random_addr(&self) -> Ipv4Addr {
-        self.range.random_client_addr()
+    pub fn random_addr(&mut self) -> Ipv4Addr {
```

Comment:
This is not really a random_addr anymore, right? You want to keep the name for compat?

Reply:
well it's more like "I don't care give me an addr". compat doesn't really matter as `&mut self` breaks stuff anyway. Better suggestions for a name?

Reply:
`new_addr`?

Reply:
We settled on unique_addr in slack